### DISCUSSION TITLE
CSV file questions

### DISCUSSION THREAD

**Discussion by Stankey** [alumet-dev/alumet#4] [Q&A] [answered]
Hello!

I have some questions about the tool and the results it provides. To add context, I am currently working on a dual processor (both Intel Xeon gold 6240) server, and I am trying to monitor the consumption of some scripts and processes that are running. I had trouble getting realistic results on other softwares (using RAPL) because I am regularly told that one process is consuming 100W to 500W, whereas the CPU is only using 1% to 3% of its capacity, and each processor has a TDP of 150W. So here I am, because you said during the GreenDays that RAPL may have issues that are not in the documentation, and maybe you could have the solutions to my problems.

First of all, to make sure that I do understand the results given in the CSV files, here is a screenshot of what I got monitoring a bash script: 
![mesure alumet](https://github.com/alumet-dev/alumet/assets/170539693/99ff7957-8deb-4a85-bb27-a494bc2f386b)

I am not sure to understand what is written on the first 17 rows, especially if I have to take the column "value" into account for the consumption calculus, because monitoring other scipts I got 0s everywhere. I get that those rows are informations about the measures below, but not much more. If you could help me with it that would be great!
For the rest of the screen, I have always 2 measures each second. I assume that for the cpu_package it means that it gives me the results for both processors, but is it normal that there is also 2 measures for the dram? I am really not a computer pro so I don't know if the dram used is linked with the processor that uses it or not, otherwise I don't know why there are two measurements.
Also, it seems that the cpu_package measurement is showing the CPU's whole consumption. Is it possible to only monitor a process to see the % of consumption using its PID for example? I kind of know where to find that information but it would be very useful in order to have a synchronised time and I would not want to skew the measures using another command or something. I just don't know if it's something that is possible with Alumet yet.

Thank you!

**Comment by TheElectronWill** [comment, marked as answer]
Hello! We've already answered you by email, so I hope that you were able to conduct your experiments :)
For other people that arrive here: the user book has been updated for every plugin, which should answer most questions.
